src/models/transactions/checkCash.ts

```diff
@@ -0,0 +1,37 @@
+import { ValidationError } from "../../common/errors";
+import { Amount, IssuedCurrencyAmount } from "../common";
+import { BaseTransaction, verifyBaseTransaction } from "./common";
+
+export interface CheckCash extends BaseTransaction {
+    TransactionType: "CheckCash";
+    CheckID: string;
+    Amount?:	Amount;
+    DeliverMin?: Amount;
```

Comment:
Amount and DeliverMin should probably be 'IssuedCurrencyAmount' instead of 'Amount' because it doesn't seem like they have the option to be a string, and the common 'Amount' type does have that option: https://xrpl.org/checkcash.html#checkcash

Reply:
It's of the type Amount which means you can transfer value using `IssuedCurrencyAmount` for non-XRP tokens and string for XRP.